scenario/utils/relayScrollMessage.ts

```diff
@@ -49,7 +49,7 @@ export default async function relayScrollMessage(
 
     const aliasAccount = await impersonateAddress(
       bridgeDeploymentManager,
-      applyL1ToL2Alias(scrollMessenger.address)
+      "0xD69c917c7F1C0a724A51c189B4A8F4F8C8E8cA0a"
```

Comment:
What address is this? Ideally, this will be pulled in from the deployment manager rather than hard-coded, since this file will be used for Scroll mainnet as well.

Reply:
For our Goerli testnet, we are using an EOA for this, which is why it's hardcoded, but for mainnet `applyL1ToL2Alias(scrollMessenger.address)` will be accurate